### DISCUSSION TITLE
Sound in Musescore goes +1 octave when disconnected from bluetooth speaker

### DISCUSSION THREAD

**Discussion by nathansuperuser** [musescore/MuseScore#25897] [Q&A] [answered]
Hi, I'm Nathan, this is my problem :
- When I connect a bluetooth speaker and then disconnect it when using Musescore the audio output from the internal speaker goes up 1 octave and it's annoying to have that, any idea of the cause?

**Comment by MarcSabatella** [comment, marked as answer]
Yes, your default audio device and the Bluetooth speaker must have different sample rates and this isn’t being communicated properly.  Either change your default audio device to match that of the Bluetooth speaker (normally 48 kHz, I think) or restart MuseScore Studio to make sure it picks up the change.

**Reply by nathansuperuser** [reply]
why do they have to use different sample rate for some reason though ?

**Reply by MarcSabatella** [reply]
That’s a question for the manufacturers of the devices.  But there are many different standards for sample rate, and different manufacturers make different choices.

**Comment by jamessmith6112** [comment]
This issue likely occurs due to a mismatch in the audio output settings when switching from a Bluetooth speaker to the device's internal speakers. Musescore may default to a different sound configuration, causing the pitch to shift. Try checking your audio output settings in Musescore and your device to ensure they match.
